README.md

```diff
@@ -210,13 +212,13 @@ The current default spec:
 Password is disabled and locked by default.
```

Comment:
Could you run [`doctoc`](https://github.com/thlorenz/doctoc) to update the TOC section few lines above?